test/tests/userobjects/openmc_tally_editor/tests

```diff
@@ -0,0 +1,130 @@
+      expect_err = 'Invalid tally score "invalid".'
+      detail = 'trying to add a score with an invalid type'
+      required_objects = 'OpenMCCellAverageProblem'
+    []
+    [nonexistent_filter]
+      type = RunException
+      input = add_filter.i
+      cli_args = 'UserObjects/new_filter/create_filter=false'
+      expect_err = 'Filter 100 does not exist and create_filter is false'
+      detail = 'the filter does not exist and is not flagged for creation'
+      required_objects = 'OpenMCCellAverageProblem'
+    []
+    [clashing_filter_types]
+      type = RunException
+      input = add_filter.i
+      cli_args = 'UserObjects/new_filter/filter_type="universe" UserObjects/new_filter/filter_id=1'
+      expect_err = 'An existing filter, Filter 1, is of type "cell" and cannot be changed to type "universe"'
+      detail = 'more than one OpenMCDomainFilterEditor eixsts with the same filter ID'
+      required_objects = 'OpenMCCellAverageProblem'
+    []
+    [clashing_filter_ids]
+      type = RunException
+      input = clashing_filter_ids.i
+      expect_err = 'Duplicate filter ID \(10\) found across multiple OpenMCDomainFilterEditors'
+      detail = 'more than one OpenMCDomainFilterEditor eixsts with the same filter ID'
+      required_objects = 'OpenMCCellAverageProblem'
+    []
+    [clashing_tally_ids]
+      type = RunException
+      input = clashing_tally_ids.i
+      expect_err = 'Duplicate tally ID \(10\) found in multiple OpenMCTallyEditors'
+      detail = 'more than one OpenMCTallyEditor eixsts with the same tally ID'
+      required_objects = 'OpenMCCellAverageProblem'
+    []
+    [clashing_mapped_tally]
+      type = RunException
+      input = clash_mapped_tally.i
+      expect_err = 'This tally ID \(3\) is a tally which Cardinal is automatically creating and controlling'
+      detail = 'an OpenMCTallyEditor eixsts for a mapped tally created by Cardinal'
+      required_objects = 'OpenMCCellAverageProblem'
+    []
+  []
+
+  [nuclides]
+    type = CheckFiles
+    input = tally_param_checks.i
+    check_files = tallies.out
+    file_expect_out ='U238'
+    cli_args = 'UserObjects/tally_editor_uo/nuclides="U238"'
+    requirement = "The system shall confine the tally to a particular nuclide. This is compared by selecting just one of the existing nuclides, and comparing to the sum of the total nuclides."
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [scatter_tally]
+    type = CheckFiles
+    input = scatter.i
+    check_files = tallies.out
+    requirement = 'Ensure that scores specified by a tally editor UO are present in the tally output'
+    file_expect_out = 'Scattering Rate'
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [absorption_u238]
+    type = CheckFiles
+    input = nuclide_absorption.i
+    check_files = tallies.out
+    requirement = 'Ensure that scores specified by a tally editor UO are present in the tally output'
```

Comment:
For SQA purposes, no two tests can have identical "requirement" tags. Can you check these two (and maybe others) and add more detail to the `requirement` so that they are not duplicates?

Reply:
All set. Sorry for the duplication/inaccurate descriptions.

Reply:
no worries, there's no way to know that unless generating the SQA documentation. Thanks!